src/explorer/specExplorer.ts

```diff
@@ -13,7 +13,7 @@ export class SpecNodeProvider implements vscode.TreeDataProvider<GaugeNode> {
         new vscode.EventEmitter<GaugeNode | undefined>();
     readonly onDidChangeTreeData: vscode.Event<GaugeNode | undefined> = this._onDidChangeTreeData.event;
 
-    constructor(private workspaceRoot: string, private languageClient: LanguageClient) {
+    constructor(private workspaceRoot: string, private lc: LanguageClient) {
```

Comment:
lets not use `golang` style short variable names in javascript. I prefer `languageClient`